### DISCUSSION TITLE
Please revive this project

### DISCUSSION THREAD

**Discussion by ntgraph** [lucidarch/lucid#54] [General]
Hi @Mulkave, please do not abandon this project.

**Comment by haralddoderer** [comment]
Had been in contact with him. I think he will be back soon.

**Reply by ntgraph** [reply]
Hope the author will come back. It is not necessary to add new features, this library serve as a way or skeleton that applications are built on top.  Needless to say it is the best architecture, still MVC but with an elegant way of dealing with codes, the more complex the more the library is valuable.  I just wish the author maintain along side each new iteration of Laravel.

**Reply by haralddoderer** [reply]
will forward the feedback :)

**Comment by Mulkave** [comment]
Hey everyone, first off, I'm sorry for the silence. Life hit hard over the past couple of years in ways I didn't anticipate, and the project unfortunately bore the cost of that.

Lucid is not dead. It still matters to me, and I'm aware of how much it matters to those of you who've built on it. I'm back to maintaining it, and with the help of AI tooling, I'm in a much better position to keep up with updates, issues, and Laravel compatibility going forward. And I will be more active in reviewing contributions, as I'm aware there are stale PRs and will address those separately.

I've already started working on the compatibility upgrades and will publish them as soon as they're ready. Thank you for your patience.

**Reply by websmurf** [reply]
Sorry to hear that life has been so bad on you in the past and it's great to see that there are updates again.
Is there any way that we can assist you in this?

**Reply by Mulkave** [reply]
Thank you, that means a lot!

The best way to help is honestly just to keep the conversation going. Share your feedback, request features, tell me what's working well and what isn't. Knowing how people are actually using Lucid and what they need from it is what shapes the direction of the project, so that kind of input is genuinely valuable.

On that note, one thing I'm thinking about next is a VS Code extension to help with unit navigation, things like go-to-feature and go-to-job. Would love to hear your thoughts on that too.
